packages/core/src/transport/eventBridge.ts

```diff
@@ -8,10 +8,22 @@ export interface DatadogEventBridge {
   send(msg: string): void
 }
 
+export type BridgeEventType = 'log' | 'view' | 'error' | 'action' | 'resource' | 'long_task'
+
+export function getEventBridge() {
+  const datadogEventBridge = getDatadogEventBridge()
+
+  return {
+    send(eventType: BridgeEventType, event: Context) {
+      datadogEventBridge?.send(JSON.stringify({ eventType, event }))
+    },
+  }
+}
+
 export function isEventBridgePresent(): boolean {
-  return !!getEventBridge()
+  return !!getDatadogEventBridge()
 }
 
-function getEventBridge() {
+function getDatadogEventBridge() {
```

Comment:
NIT: maybe `getEventBridgeGlobal` would be more explicit to avoid naming clash with `getEventBridge`